data/avro-schema/src/main/resources/avro/net/corda/data/virtualnode/EntityResponseFailure.avsc

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "record",
+  "name": "EntityResponseFailure",
+  "namespace": "net.corda.data.virtualnode",
+  "fields": [
+    {
+      "name": "errorType",
+      "doc": "Error type",
+      "type": {
+        "name": "Error",
+        "type": "enum",
+        "doc": "FATAL: almost certainly unrecoverable.\nNOT_READY: the component has not yet started or received all cpks.\nDATABASE: an error during database execution.\nVIRTUAL_NODE: problems retrieving vnode info\nUNKNOWN_COMMAND: an unhandled request payload type",
+        "symbols": [
+          "FATAL",
+          "NOT_READY",
+          "DATABASE",
+          "VIRTUAL_NODE",
+          "UNKNOWN_COMMAND"
+        ]
```

Comment:
I'm fine with starting with this. We might find we can't do very much with the distinctions between the non-fatal errors, but again, this feels like something we can adjust later.

Reply:
"worst" case with code is:

```
if (errorType == Error.FATAL)  abortFlowWorker() else retry()
```